test/e2e/framework/auth/helpers.go

```diff
@@ -57,12 +63,12 @@ func WaitForNamedAuthorizationUpdate(c v1beta1authorization.SubjectAccessReviews
 			User: user,
 		},
 	}
+
 	err := wait.Poll(policyCachePollInterval, policyCachePollTimeout, func() (bool, error) {
 		response, err := c.SubjectAccessReviews().Create(review)
 		// GKE doesn't enable the SAR endpoint.  Without this endpoint, we cannot determine if the policy engine
 		// has adjusted as expected.  In this case, simply wait one second and hope it's up to date
-		if apierrors.IsNotFound(err) {
-			klog.Info("SubjectAccessReview endpoint is missing")
+		if apierrors.IsNotFound(err) && framework.ProviderIs("gke") {
```

Comment:
This is the _only_ line that ties this package to the framework. Considering how hard it is to decouple the testContext/framework/etc at this time, I think we should just leave this as-is with a FIXME line.